packages/components/src/dropdown/README.md

```diff
@@ -90,13 +90,6 @@ Use this object to access properties/features of the `Popover` component that ar
 
 -   Required: No
 
-### `position`: `PopoverProps[ 'position' ]`
-
-The direction in which the popover should open relative to its parent node. Specify a y- and an x-axis as a space-separated string. Supports `"top"`, `"bottom"` y-axis, and `"left"`, `"center"`, `"right"` x-axis.
-
--   Required: No
--   Default: `"top center"`
```

Comment:
I'm not sure that the documented default value for `position` was correct — the `position` prop didn't seem to have any default value assigned, and even the prop's default value in the `Popover` component is different